hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -433,6 +433,208 @@
           ]
         }
       },
+      {
+        "transactionFeeSchedule": {
+          "hederaFunctionality": "TokenAirdrop",
```

Comment:
Do we need all these types? We should just have a default case. Since any type of airdrop costs only $0.001 right ?

Reply:
@Neeharika-Sompalli fixed
@povolev15 can you give me a link to the change? I'm not quite sure what you mean by using a constant.